docs/code-snippets/outlook-snippets.yaml

```diff
@@ -1346,10 +1346,18 @@ Office.Recurrence.getAsync:
     function callback(asyncResult){
         var context = asyncResult.context;
         var recurrence = asyncResult.value;
-        var endDate = recurrence.seriesTime.getEndDate();
+        if (recurrence == null) {
+            console.log("Non-recurring meeting");
+            return;
```

Comment:
Instead of returning, could this be made into an if/else statement?

Reply:
Yes, we could add an else. Thoughts from the team about if/return vs if/else? Michelle was also wondering about == vs === - thoughts about that? (Do we have some coding guidelines we want to follow?)

Reply:
I know there's a greater coding principle around minimizing exit paths from methods. In this case, you're not returning because of an error state, but an expected condition. You're logically performing the if/else anyway, but it doesn't appear that way at first glance.

Reply:
@AlexJerabek, @mscharlock: I'll put coding guidelines on the team agenda and we can discuss these specific examples then.